network/proxy.md

```diff
@@ -4,6 +4,12 @@ description: How to configure the Docker client to use a proxy server
 keywords: network, networking, proxy, client
 ---
 
+> **Note**
+>
+> This page describes how to configure the Docker CLI to configure proxies via environment variables in containers.
+> For information on configuring Docker Desktop to use HTTP/HTTPS proxies, see [proxies on Mac](../desktop/mac/index.md#proxies) and [proxies on Windows](../desktop/windows/index.md#proxies).
```

Comment:
Also need to add the DD4L link; https://docs.docker.com/desktop/linux/#proxies

That said;

- Perhaps the "proxies" section should be in a general page (looks like it's effectively the same for all variants)
- Currently the linked to section only describes "To configure HTTP proxies, switch on the Manual proxy configuration setting.", then continues to show a single example with `docker run -e ...`, and links back to this page for details (so we'd be sending users in a loop 😂); perhaps the `docker run -e` example should be removed there, and this page improved a bit.

Reply:
Thanks @thaJeztah. Added your suggestion. @aevesdocker is working on consolidating the Desktop docs. Adding a generic proxy page will be part of this effort. We can also do a follow-up PR for further improvements to the proxy page and to update based on your suggestion on adding a single example with `docker run -e`.